src/Entity/Swap.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace App\Entity;
+
+use App\Entity\User;
+use App\Entity\Skill;
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+use App\Repository\SwapRepository;
+
+/**
+ * @ORM\Entity(repositoryClass=SwapRepository::class)
+ */
+class Swap
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private int $id;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=User::class, inversedBy="askedSwaps")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private ?User $asker;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=User::class, inversedBy="helpedSwaps")
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private ?User $helper;
+
+    /**
+     * @ORM\ManyToOne(targetEntity=Skill::class)
+     * @ORM\JoinColumn(nullable=false)
+     */
+    private ?Skill $skill;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private \DateTimeInterface $date;
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private string $message;
```

Comment:
You probably should have constraints to validate the message... May be avoiding empty messages ?